send_cleared_status_to_person_hub.R

```diff
@@ -0,0 +1,30 @@
+library(tidyverse)
+library(dotenv)
+library(REDCapR)
+library(lubridate)
+
+source("functions.R")
+
+script_run_time <- with_tz(now(), tzone = Sys.getenv("TIME_ZONE"))
+Sys.getenv("INSTANCE")
+
+# fetch data from REDCap
+fields <- c('record_id', 'cleared_for_campus', 'cleared_for_campus_timestamp')
+redcap_results_rows <- redcap_read(
+        redcap_uri = Sys.getenv("URI"),
+        token = Sys.getenv("RESULT_TOKEN"),
+        batch_size = 1000L,
+        fields = fields,
+        guess_type=F, # force all columns to strings
+        guess_max = 3000)$data %>%
+        filter(!is.na(cleared_for_campus)) %>%
+        select(UFID = record_id, CLEARED_FOR_CAMPUS = cleared_for_campus, CLEARED_FOR_CAMPUS_TIMESTAMP = cleared_for_campus_timestamp)
+
+# sanity check there are no redundant ufids
+redcap_results_rows$ufid %>% unique() %>% length() == redcap_results_rows %>% nrow()
```

Comment:
```suggestion
redcap_results_rows$UFID %>% unique() %>% length() == redcap_results_rows %>% nrow()
```
the upcase goofed this one line.